frontend/src/screens/learner.tsx

```diff
@@ -167,9 +169,34 @@ export const MobileStudyCards: FC<{studies: ParticipantStudy[]}> = ({ studies })
 }
 
 export const DesktopStudyCards: FC<{studies: ParticipantStudy[]}> = ({ studies }) => {
+
+    const [hovered, setHovered] = useState(-1)
+    const [displayArrows, setDisplayArrows] = useState<boolean>(false)
+    const [element, setElement] = useState<HTMLDivElement | null>(null)
+
+    const [isOverflowing, setIsOverflowing] = useState(false);
+
+    useEffect(() => {
+        const checkOverflow = () => {
+            if (element) {
+                const overflow = element.scrollWidth > element.clientWidth;
+                setIsOverflowing(overflow);
+            }
+        };
+
+        checkOverflow();
+
+        window.addEventListener('resize', checkOverflow);
+        return () => window.removeEventListener('resize', checkOverflow);
+    }, [studies, element]);
+
     return (
-        <Box>
-            <Swiper
+        <Box style={{ position: 'relative', paddingLeft: '3rem', paddingRight: '3rem' }} onMouseOver={() => {
+            setDisplayArrows(true)
+        }} onMouseLeave={() => {
+            setDisplayArrows(false)
+        }}>
+            {/* <Swiper
```

Comment:
commented code can probably get removed right?

Reply:
Yeah It can be removed, I kept it just in case